### PR TITLE
feat(swagger): REST API for registration

### PR DESCRIPTION
**Pull request by leopardilla**
## WHAT

updated swagger for registration

related to #74

### CODE REVIEW

File: backend/server/static/api/meetingbook.yaml [hunk 1]
```diff
@@ -109,6 +133,19 @@ paths:
           schema:
             $ref: "#/definitions/Booking"
 definitions:
+  RegistrationBody:
+    type: "object"
+    required:
+      - email
+      - password
+      - password_confirm
```
Comment (mentortechabc) [inline backend/server/static/api/meetingbook.yaml:141]: discussed in chat. 

I'd propose to remove password_confirm from the api

**Review by cornhubius — CHANGES_REQUESTED** [review]

File: backend/server/static/api/meetingbook.yaml [hunk 2]
```diff
@@ -37,6 +37,30 @@ paths:
...
         "200":
           description: "Successful admin request"
+  /registration:
+    post:
+      summary: Register a new Admin
+      description: Requires a new email (not yet registered in MeetingBook), password and the second input of this password to confirm the registration of a new Admin.
+      consumes:
+        - application/json
+      parameters:
+        - in: body
```
[the remark this answers is not in the document]
Comment (mentortechabc) [reply backend/server/static/api/meetingbook.yaml:47]: good catch

File: backend/server/static/api/meetingbook.yaml (hunk 1 above)
Comment (leopardilla) [reply backend/server/static/api/meetingbook.yaml]: fixed

File: backend/server/static/api/meetingbook.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (leopardilla) [reply backend/server/static/api/meetingbook.yaml:47]: fixed

**Review by mentortechabc — APPROVED** [review]

**Review by cornhubius — APPROVED** [review]

**Pull request merged**